src/directives/mobilegeolocation.js

```diff
@@ -0,0 +1,199 @@
+goog.provide('ngeo.MobileGeolocationController');
+goog.provide('ngeo.mobileGeolocationDirective');
+
+goog.require('ngeo');
+goog.require('ngeo.DecorateGeolocation');
+goog.require('ngeo.FeatureOverlay');
+goog.require('ngeo.FeatureOverlayMgr');
+goog.require('ol.Feature');
+goog.require('ol.Geolocation');
+goog.require('ol.Map');
+goog.require('ol.geom.Point');
+goog.require('ol.proj');
+
+
+/**
+ * Provide a "mobile geolocation" directive.
+ *
+ * @example
+ * <button class="ngeo-mobile-geolocation" ngeo-mobile-geolocation="::"
+ *   ngeo-mobile-geolocation-map="ctrl.map"
+ *   ngeo-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
+ * </button>
+ *
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname ngeoMobileGeolocation
+ */
+ngeo.mobileGeolocationDirective = function() {
+  return {
+    restrict: 'A',
+    scope: {
+      'getMobileMapFn': '&ngeoMobileGeolocationMap',
+      'getMobileGeolocationOptionsFn': '&ngeoMobileGeolocationOptions'
+    },
+    controller: 'NgeoMobileGeolocationController',
+    controllerAs: 'ctrl',
+    template: undefined
```

Comment:
Remove this line, it put that in the doc to focus that there is no template, but the line is useless.
I will update the doc to avoid confusion, sorry for that.